src/test/java/com/ericsson/ei/subscription/SubscriptionValidatorTest.java

```diff
@@ -247,11 +235,12 @@ public void validateNotificationMessageInvalidMessage3Test() throws Exception {
         subscription.getNotificationMessageKeyValues().add(notificationMessageKeyValue);
         try {
             invokeMethod(SubscriptionValidator.class, "validateNotificationMessageKeyValues",
-                    subscription.getNotificationMessageKeyValues(), MediaType.APPLICATION_FORM_URLENCODED.toString());
+                    subscription.getNotificationMessageKeyValues(),
+                    MediaType.APPLICATION_FORM_URLENCODED.toString());
+            fail("Expected subscriptionvalidation to throw exception.");
```

Comment:
Here and elsewhere why not use `@Test(expected = SubscriptionValidationException.class)`

Reply:
Maybe since expected exceptions may be tested, but updating old tests is not prioritized.

Reply:
The question should then be: Why add a fail when you can use the test expection